### DISCUSSION TITLE
Getting release number from git in command line

### DISCUSSION THREAD

**Discussion by anjanesh** [community/community#21593] [New to GitHub]
<pre><code class="lang-auto">$ git --git-dir=.git --work-tree=projectName describe --abbrev=0 --tags
release-1.5.5
</code></pre>
<p>Is there someway to get the exact release number 1.5.5 from git or do I need to depend on string manipulation to achieve that ?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>That command gives you the most recent tag reachable from the current commit. Git itself doesn’t ascribe any meaning to tag names, you’ll need to handle that yourself. So yes, string manipulation it is.</p>
